### ISSUE TITLE
Cannot divide evenly which means cannot form batches

### ISSUE THREAD

**Issue by Ziksby** [Ziksby/MetaLearnCuriosity#12]
**Bug Description:**

When attempting to create a batch we run into an issue whereby we cannot divide evenly the shapes 128 and 512 when we do not times the intrinsic reward by `(1-done)`.

**Steps to Reproduce:**

1. Do not times the intrinsic reward by `(1-done)` in BYOL explore.
2. Then run the BYOL-explore file.

**Expected Behavior:**

We should expect the file to execute and we should be able to run the experiments.

**Actual Behavior:**

We obtain this error:
`jax._src.core.InconclusiveDimensionOperation: Cannot divide evenly the sizes of shapes (128,) and (512,)`

**Environment:**

Empty-misc

**Additional Information:**

`    config = {
        "SEED": 42,
        "NUM_SEEDS": 30,
        "LR": 2.5e-4,
        "NUM_ENVS": 4,
        "NUM_STEPS": 128,
        "TOTAL_TIMESTEPS": 5e5,
        "UPDATE_EPOCHS": 4,
        "NUM_MINIBATCHES": 4,
        "GAMMA": 0.99,
        "GAE_LAMBDA": 0.95,
        "CLIP_EPS": 0.2,
        "ENT_COEF": 0.01,
        "VF_COEF": 0.5,
        "MAX_GRAD_NORM": 0.5,
        "ACTIVATION": "tanh",
        "ENV_NAME": "Empty-misc",
        "ANNEAL_LR": True,
        "DEBUG": False,
        "EMA_PARAMETER": 0.99,
        "REW_NORM_PARAMETER": 0.99,
        "INT_LAMBDA": 0.1,
    }`
